docs/commands.md

```diff
@@ -546,3 +547,48 @@ Upgrade your app's template files to the specified or latest npm version using [
 Using this command is a recommended way of upgrading relatively simple React Native apps with not too many native libraries linked. The more iOS and Android build files are modified, the higher chance for a conflicts. The command will guide you on how to continue upgrade process manually in case of failure.
 
 _Note: If you'd like to upgrade using this method from React Native version lower than 0.59.0, you may use a standalone version of this CLI: `npx @react-native-community/cli upgrade`._
+
+### `profile-hermes`
+
+Usage:
+
+```sh
+react-native profile-hermes [destinationDir] <flag>
+```
+
+Pull and convert a Hermes tracing profile to Chrome tracing profile, then store it in the directory <destinationDir> of the local machine.
+
+- `destinationDir` is optional, if provided, pull the file to that directory
+  > default: pull to the current React Native app root directory
+
+#### Options
+
+#### `--fileName [string]`
```

Comment:
`--fileName` option name is inconsistent with other options.
I think it should be `--filename` or `--file-name`

Reply:
Thanks, I changed it accordingly!